addons/account/models/account_move.py

```diff
@@ -3916,6 +3916,11 @@ def check_full_reconcile(self):
                     exchange_move_id = exchange_move.id
                 else:
                     missing_exchange_difference = True
+            elif to_balance:
+                for aml_to_balance, total in to_balance.values():
+                    if not total:
```

Comment:
I think this will create other issue.

The lines aren't reconcile because there is a remaining `amount_residual` or `amount_residual_currency` and, maybe I'm wrong, I think it's intended.

You should probably handle it by creating a special case for the full refund instead.
Maybe @smetl could give more insight about what should be done.

Reply:
Aren't we supposed to fix stuff that were raised before the end of support of the version?